eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,34 @@
+$Language = "android"
+
+function Get-AllPackageInfoFromRepo ($serviceDirectory)
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+
+  if ($serviceDirectory)
+  {
+    $properties = gradle ":sdk:$serviceDirectory:retrieveProjectProperties" -q
```

Comment:
I'm not familiar with gradle conventions can you link to the resources you used to learn about how this works?

Reply:
We have some settings defined in https://github.com/Azure/azure-sdk-for-android/blob/main/settings.gradle for the multi project build. I figured out by trial and error that the one can run a gradle task command for the the specific package, the service or the entire repo.